templates/serverpod_templates/projectname_server_upgrade/docker-compose.yaml

```diff
@@ -11,12 +12,45 @@ services:
       POSTGRES_PASSWORD: "DB_PASSWORD"
     volumes:
       - projectname_data:/var/lib/postgresql/data
+    profiles:
+      - '' # Default profile
```

Comment:
Question: For my information - How did this work with the default profile?

Reply:
docker compose treats empty profile as the "default", i.e. it will run if you simply type
```bash
docker compose up
```
If you don't have a "default" profile, then you have to explicitly add `--profile test` for that service to be executed.

Reply:
Aah thank you! 🌟